creator-node/test/resizeImage.test.js

```diff
@@ -66,7 +66,7 @@ describe('test resizeImage', () => {
    * Then: an error is thrown
    */
   it('should not throw error if ipfs is down', async () => {
```

Comment:
lets rename this test to `should not throw if generating ipfs multihash fails`

Reply:
done